tests/dummy/app/styles/defaults/_examples.scss

```diff
@@ -0,0 +1,42 @@
+@import '../variables';
+
+.example {
+  centered {
+    background-color: $theme-color-1;
+    &:nth-child(even) {
+      background-color: $theme-color-2;
+    }
+  }
+}
+
+hbox.header {
+  height: 50px;
+}
+
+.blue {
+  background-color: $theme-color-1;
+}
+
+.fixed-250 {
+  width: 250px;
```

Comment:
We don't want the sidebar column to move though, in a UI scenario it's not ideal. It could have a breakpoint sure, but not flex it's width. Maybe there's a better way to express this instead of a fixed number though.